app/src/main/res/values/strings.xml

```diff
@@ -1311,6 +1311,7 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_sidewalk_value_yes">sidewalk</string>
     <string name="quest_sidewalk_value_no">no sidewalk</string>
     <string name="quest_sidewalk_value_separate">sidewalk, but displayed separately on map</string>
+    <string name="quest_sidewalk_answer_different">The sidewalk situation is different</string>
```

Comment:
```suggestion
    <string name="quest_sidewalk_answer_different">Sidewalk situation is different</string>
```

Shorter text because space in popup is limited